src/spell/modules/spell/spell/src/log.rs

```diff
@@ -129,7 +129,7 @@ mod tests {
         let store = spell.store_log_cp(log1.clone(), cp.clone());
         assert!(store.success, "{}", store.error);
 
-        std::thread::sleep(std::time::Duration::from_secs(1));
+        std::thread::sleep(std::time::Duration::from_millis(1500));
```

Comment:
Why is sleep needed? 🤔

Reply:
I want to test timestamps by sqlite

Reply:
Why sleep for so long then? 1.5 seconds is a lot of time for tests, quickly adds up

Reply:
Timestamps is in seconds, I can remove this check at all, if you'd like